elastic/service.go

```diff
@@ -464,6 +469,8 @@ func (s *service) additionalWhere(dataSourceType, sortField string) (string, err
 			return `and \"time_to_close_days\" is not null`, nil
 		case "jira_comments":
 			return `and \"comment_id\" is not null and \"type\" = 'comment'`, nil
+		case "jira_issues_closed":
+			return `and \"assignee_uuid\" is not null and \"status\" = 'Closed' or \"status\" = 'Resolved' or \"status\" = 'Done'`, nil
```

Comment:
If you use `or` condition - it should be in `(` imho `)` becaus eall those conditions are merged together (`ANDed`) to form a final condition.

Reply:
Actually, I would give `and \"status\" in ('Closed', 'Resolved', 'Done')`

Reply:
fixed